tests/platform_tests/conftest.py

```diff
@@ -19,7 +19,7 @@
 
 TEMPLATES_DIR = os.path.join(os.path.dirname(
     os.path.realpath(__file__)), "templates")
-FMT = "%b %d %H:%M:%S.%f"
+FMT = "%Y %b %d %H:%M:%S.%f"
```

Comment:
I think the existing format still needs to be supported, since there could be syslogs from an older existing image that need to be successfully parsed. This would mean that the new format would be an additional format to test the log timestamp against.

Reply:
Updated in latest commit.